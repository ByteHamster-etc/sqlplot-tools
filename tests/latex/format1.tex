line
%% TABULAR REFORMAT(round=floor)
%% SELECT 0.001, 0.01, 0.1, 1, 10.1, 100.1, 1000.1
line
%% TABULAR REFORMAT(round=ceil)
%% SELECT 0.001, 0.01, 0.1, 1, 10.1, 100.1, 1000.1
line
%% TABULAR REFORMAT(round=0)
%% SELECT 0.001, 0.01, 0.1, 1, 10.1, 100.1, 1000.1
line
%% TABULAR REFORMAT(round=1)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(round=-1)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(precision=3)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(col 2,4=(precision=3))
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(digits=3)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(group precision=1)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(group=(!) precision=1)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(col 2=(max=bold) col 3-4=(min=emph))
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(row 0=(max=bold))
%% SELECT 0.001, 0.01, 0.1, 1, 1000.05, 10.04, 100.05
line
%% TABULAR REFORMAT(row 1=(max=bold))
%% SELECT 0.001, 0.01, 0.1, 1, 1000.05, 10.04, 100.05
end