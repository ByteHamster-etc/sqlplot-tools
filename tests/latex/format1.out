line
%% TABULAR REFORMAT(round=floor)
%% SELECT 0.001, 0.01, 0.1, 1, 10.1, 100.1, 1000.1
0 & 0 & 0 & 1 & 10 & 100 & 1000 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.1, 100.1, 1000.1
line
%% TABULAR REFORMAT(round=ceil)
%% SELECT 0.001, 0.01, 0.1, 1, 10.1, 100.1, 1000.1
1 & 1 & 1 & 1 & 11 & 101 & 1001 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.1, 100.1, 1000.1
line
%% TABULAR REFORMAT(round=0)
%% SELECT 0.001, 0.01, 0.1, 1, 10.1, 100.1, 1000.1
0 & 0 & 0 & 1 & 10 & 100 & 1000 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.1, 100.1, 1000.1
line
%% TABULAR REFORMAT(round=1)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
0.0 & 0.0 & 0.1 & 1.0 & 10.0 & 100.1 & 1000.1 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(round=-1)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
0 & 0 & 0 & 0 & 10 & 100 & 1000 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(precision=3)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
0.001 & 0.010 & 0.100 & 1.000 & 10.040 & 100.050 & 1000.050 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(col 2,4=(precision=3))
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
0.001 & 0.01 & 0.100 & 1 & 10.040 & 100.05 & 1000.05 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(digits=3)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
0.001 & 0.010 & 0.100 & 1.00 & 10.0 & 100 & 1000 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(group precision=1)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
0.0 & 0.0 & 0.1 & 1.0 & 10.0 & 100.0 & 1,000.0 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(group=(!) precision=1)
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
0.0 & 0.0 & 0.1 & 1.0 & 10.0 & 100.0 & 1!000.0 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(col 2=(max=bold) col 3-4=(min=emph))
%% SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
0.001 & 0.01 & \bf 0.1 & \em 1 & \em 10.04 & 100.05 & 1000.05 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 10.04, 100.05, 1000.05
line
%% TABULAR REFORMAT(row 0=(max=bold))
%% SELECT 0.001, 0.01, 0.1, 1, 1000.05, 10.04, 100.05
0.001 & 0.01 & 0.1 & 1 & \bf 1000.05 & 10.04 & 100.05 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 1000.05, 10.04, 100.05
line
%% TABULAR REFORMAT(row 1=(max=bold))
%% SELECT 0.001, 0.01, 0.1, 1, 1000.05, 10.04, 100.05
0.001 & 0.01 & 0.1 & 1 & 1000.05 & 10.04 & 100.05 \\
% END TABULAR SELECT 0.001, 0.01, 0.1, 1, 1000.05, 10.04, 100.05
this is the end
